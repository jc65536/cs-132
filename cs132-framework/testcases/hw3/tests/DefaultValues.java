class Main {
	public static void main(String[] a){
		int x;
        boolean y;
        A b;
        A c;
        b = new A();
        System.out.println(x);
        System.out.println(y);
        System.out.println(b.run());
        System.out.println(c);
        System.out.println(c.run());
	}
}

class A 
{
    int x;
    boolean y;
    A a;

    public int run()
    {
        System.out.println(y);
        System.out.println(a);
        return x;
    }
}