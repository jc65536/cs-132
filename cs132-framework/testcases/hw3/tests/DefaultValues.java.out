0
0
0
0
0
0
--- Error (Explicit) ---
null pointer
